pkgs/tools/misc/logtop/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, ncurses, uthash, pkg-config }:
+
+stdenv.mkDerivation {
+  name = "logtop-0.7";
+
+  src = fetchFromGitHub {
+    rev = "a0935ab2e7056feb3e8a90f5129990c9b335a587";
+    owner = "JulienPalard";
+    repo  ="logtop";
+    sha256 = "1f8vk9gybldxvc0kwz38jxmwvzwangsvlfslpsx8zf04nvbkqi12";
+  };
+
+  buildInputs = [ncurses uthash pkg-config];
```

Comment:
`pkg-config` is supposed to be in `nativeBuildInputs`. Those are basically dependencies that are not needed for the compiled result, but are executed during the build itself. This is needed for e.g. cross-builds where the build is run on another architecture than the one the package is compiled for. Further details can be found in https://nixos.org/manual/nixpkgs/stable/#ssec-stdenv-dependencies